glean-core/src/database/mod.rs

```diff
@@ -44,6 +50,91 @@ mod backend {
     pub fn rkv_new(path: &Path) -> Result<Rkv, rkv::StoreError> {
         Rkv::new::<rkv::backend::SafeMode>(path)
     }
+
+    fn delete_and_log(path: &Path, msg: &str) {
+        if let Err(err) = fs::remove_file(path) {
+            match err.kind() {
+                std::io::ErrorKind::NotFound => {
+                    // silently drop this error, the file was already non-existing
+                }
+                _ => log::warn!("{}", msg),
+            }
+        }
+    }
+
+    fn delete_lmdb_database(path: &Path) {
+        let datamdb = path.join("data.mdb");
```

Comment:
Do we need to adjust or fix [this](https://searchfox.org/glean/rev/dc60dc2d370fe0107f959bc74de10c3e0ddfce98/glean-core/src/database/mod.rs#123)?

Reply:
We will have "wrong" data exactly once (on the first start that migrates): database size will be 0 (because it checks the wrong file).
#1304 should fix this though.